tests/test_plugin.py

```diff
@@ -151,51 +154,54 @@ def get_result(self):
             'case_id': 1234,
             'status_id': TESTRAIL_TEST_STATUS["failed"],
             'comment': "An error",
-            'duration': 2
+            'duration': 2,
+            'defects': None
         },
         {
             'case_id': 5678,
             'status_id': TESTRAIL_TEST_STATUS["failed"],
             'comment': "An error",
-            'duration': 2
+            'duration': 2,
+            'defects': None
```

Comment:
maybe you can add tests that have defects?

Reply:
Yeah, you're right.
I've just added some meaningful defects value.
Thanks Peter